.github/workflows/build.yml

```diff
@@ -73,8 +78,14 @@ jobs:
         run: |
           echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
       - name: build
+        if: startsWith(matrix.os, 'ubuntu')
         run: make build
-        if: "env.GIT_DIFF != '' || github.event_name == 'issue_comment'"
```

Comment:
if this condition isn't there, it may break the "/runsim" simulation workflow -- @allthatjazzleo may comment?

perhaps for windows/Mac -- instead of changing "build.yml", create a separate workflow ?

Reply:
This line shall not break "/runsim", it was to make sure the workflow will run only when there are some changes in `.go .mod .sum` file.
But since `technote-space/get-diff-action` step will empty result when event is `issue_comment`. I needed to put the second condition for event triggered by `/runsim` manually.

Reply:
restored back , if condtions